generate/requiredParams.go

```diff
@@ -49,7 +49,11 @@ var requiredParams = map[string][]string{
 		"displaytext",
 	},
 	"createVPC": {
+		"name",
 		"displaytext",
+		"vpcofferingid",
+		"zoneid",
+		"cidr",
```

Comment:
actually only need to add `cidr` 
but no issues if add others (`name`, `zoneid`, `vpcofferingid`)